src/template.rs

```diff
@@ -14,16 +15,51 @@ pub struct Template {
 impl Template {
     /// Renders an instance of the Template, using the given globals.
     pub fn render(&self, globals: &dyn crate::ObjectView) -> Result<String> {
+        self.render_with_mode(globals, RenderingMode::Strict)
+    }
+
+    /// Renders an instance of the Template, using the given globals.
+    pub fn render_to(&self, writer: &mut dyn Write, globals: &dyn crate::ObjectView) -> Result<()> {
+        self.render_to_with_mode(writer, globals, RenderingMode::Strict)
+    }
+
+    /// Renders an instance of the Template, using the given globals in lax mode.
+    pub fn render_lax(&self, globals: &dyn crate::ObjectView) -> Result<String> {
+        self.render_with_mode(globals, RenderingMode::Lax)
+    }
+
```

Comment:
Rather than having a bunch of different render funtions, should this be stored with the Parser and passed in when this is created?

Reply:
I actually originally started that route but decided against it.

When stored on the parser that means you're unable to determine which mode you'd want to use after the parser is created.  Every template that's parsed would be stuck in that rendering mode.  People would in theory, need to then create two separate parsers to work around that limitation.  Admittedly, I'm not sure how frequent people would ever need to change the rendering mode of a parser. I don't personally have a use case to argue that.

Another option is we could expose a function to set the rendering mode on the `Template` instead.  I'm not convinced that's a better UX, but probably a bit less confusing to most users who may use this solely in "strict" mode.